Modules/posixmodule.c

```diff
@@ -176,7 +176,9 @@ corresponding Unix manual entries for more information on calls.");
 #else
 /* Unix functions that the configure script doesn't check for */
 #define HAVE_EXECV      1
+#ifndef __ANDROID__
```

Comment:
Isn't there a way to check the Android API level at build time?  Per your comments on the issue, when running on P (API 28) it should be available.

If not, at least mention that P (API 28) contains the API in a comment here so someone can revisit this in the future.

Reply:
Technically the macro ```__ANDROID_API__``` can be used, while I prefer to wait until Android P is publicly available so that test_posix_spawn can be run before enabling it.

Reply:
I'm fine with always skipping, and "revisit" the code once API 28 is released.